src/go/gcsrunner/transform_config.go

```diff
@@ -39,13 +39,20 @@ var (
 )
 
 func addGCPAttributes(cfg *scpb.FilterConfig, opts FetchConfigOptions) error {
+	var overridePlatform string
+	if oldAttrs := cfg.GetGcpAttributes(); oldAttrs != nil {
+		overridePlatform = oldAttrs.GetPlatform()
+	}
 	co := options.DefaultCommonOptions()
 	co.MetadataURL = opts.MetadataURL
 	mf := metadata.NewMetadataFetcher(co)
 	attrs, err := mf.FetchGCPAttributes()
 	if err != nil {
 		return err
 	}
+	if overridePlatform != "" {
```

Comment:
This seems buggy.   How about we define a separate field "platform_override" in service_control filter config. GCPAttributes are always fetched from Metadata.   Let c++ code to do the platform override.

Reply:
As was pointed out above:
https://github.com/GoogleCloudPlatform/esp-v2/pull/45#discussion_r387839265

this platform override field is already a feature of the bootstrap. This change will just stop GCS Runner from ignoring that override with this one. It might make sense in the future to update Envoy by changing this implementation, but for now I was hoping to unblock our use of the feature to get metrics asap.

Reply:
Ok, then.